WikiWatershed/model-my-watershed
# Update Logo, Favicon

## Overview

Updates logo and favicon to latest.

Connects #3011 

### Demo

![image](https://user-images.githubusercontent.com/1430060/48024147-d4740d80-e10d-11e8-81db-b471d934b27e.png)

## Testing Instructions

* Check out this branch
* Run `./scripts/bundle.sh --vendor`
* Run `./scripts/manage.sh collectstatic`
* Go to [:8000/](http://localhost:8000/) and hard refresh
* Ensure you see new favicon and logo in the top-left
